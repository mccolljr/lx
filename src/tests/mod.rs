use std::process::Command;

#[test]
fn test_snapshots() {
    glob!("testdata/*.lx", |path| {
        println!("testing {:?}", path);
        let output = Command::new("cargo")
            .arg("run")
            .arg(path)
            .output()
            .expect("failed to execute test");
        let output_str =
            String::from_utf8(output.stdout).expect("oops, output not utf8");
        assert_snapshot!(output_str);
    });
}
