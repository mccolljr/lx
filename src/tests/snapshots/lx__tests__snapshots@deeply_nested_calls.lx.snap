---
source: src/tests/mod.rs
expression: output_str
input_file: src/tests/testdata/deeply_nested_calls.lx
---
got: Int(155):
OK!

